modules/commands/outs.py

```diff
@@ -54,13 +55,27 @@ def send(self, method):
                 margin=STR_MORE_LEN)
 
             if truncated:
+                valid, truncated = self._adjust_to_word_boundaries(valid, truncated)
+
                 line = utils.irc.parse_line(valid+STR_MORE)
                 self._text = "%s%s" % (STR_CONTINUED, truncated)
             else:
                 self._text = ""
 
             sent_line = self.server.send(line)
 
+    @staticmethod
+    def _adjust_to_word_boundaries(left, right):
```

Comment:
please remove `@staticmethod` and just make the func def `def _word_boundary_split(self, left, right):`

Reply:
Done :smiley: